docs/source/install.rst

```diff
@@ -4,6 +4,89 @@
 Installing Volttron
 ===================
 
+Installing Linux Virtual Machine
+--------------------------------
+Volttron requires a Linux system to run. For most this will require virtual machine (VM). This section will describe the
+steps necessary to install VOLTTRON using Oracle VirtualBox software. Virtual Box is free and can be
+downloaded from https://www.virtualbox.org/wiki/Downloads.
+
+|VirtualBox Download|
+
+.. |VirtualBox Download| image:: images/vbox-download.png
+
+After installing VirtualBox, it will also be necessary to download a Linux operating system image for use on the VM.
+Ubuntu 14.04 LTS or Linux Mint 17 or later is the recommended Linux operating system for use with VOLTTRON, however any
+distribution will work. Linux Mint 18.3 with the Xfce desktop is used as an example, however platform setup in Ubuntu
+should be identical. A 32-bit version of Linux should be used when running VOLTTRON on a system with limited hardware
+(less than 2 GB of RAM). Go to https://www.osboxes.org/linux-mint/ to download one of the versions of Linux Mint and
+extract VDI from downloaded archive.
+
+Note: this is a prepackaged version with many of the VM settings already defined. To get the base version of Linux Mint
+go to https://linuxmint.com/.
```

Comment:
I don't think this note belongs at all.

Reply:
I asked Jereme about referring to the main Linux Mint website the other day. He said "A short note about where to find the base Mint would be good." Is there any place in this where it could belong? Or does it just not work?

Reply:
So up above where you specify the iso or whatever you could have a list of different distributions in a box if you wanted to do that...